### PR TITLE
Server

### PR DESCRIPTION
**Pull request by kmorgan8588**
First PR, practiced making a feature branch. Sets up basic structure of file system for server/db

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+const URI = 'mongodb://localhost/listings';
```
Comment (inkaivasyuk) [inline db/index.js]: Should you maybe include the port number?

- "mongodb://localhost:27017/test" 

I think I remember, Matt, mentioning that we might want to include it.

Comment (kmorgan8588) [reply db/index.js]: Good idea, let me add that. Thanks!

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+app.use(express.json());
+
+
+app.route('/api/listings')
+    .get((req, res) => {
+        console.log(req.url);
+        // console.log(House);
+        // res.sendStatus(200);
```
Comment (inkaivasyuk) [inline server/index.js:14]: Would be good to get rid of console.logs and unnecessary comments.

**Review by inkaivasyuk — CHANGES_REQUESTED** [review]
Looks good to me, just minor feedback.

File: server/index.js (hunk 2 above)
Comment (kmorgan8588) [reply server/index.js:14]: Oh yeah, that's leftover from my postman tests

**Review by inkaivasyuk — APPROVED** [review]
Looks good.

**Pull request merged**